core/src/main/scripts/importer/cbioportal_common.py

```diff
@@ -554,8 +554,22 @@ def flush(self):
                                  record.getMessage())
             if identifying_tuple not in grouping_dict:
                 grouping_dict[identifying_tuple] = []
-            grouping_dict[identifying_tuple].append(record)
 
+            #Add a tuple that keeps the relation between the line number,
+            #the column number and the cause for the HTML report
+            line_number = None
+            column_number = None
+            cause = None
+            if 'line_number' in record.__dict__:
```

Comment:
This block of code looks overcomplicated, talking about internal instance dicts and looking for the attribute twice; I'd recommend making the triple in this way instead:
```python
(
    getattr(record, 'line_number', None),
    getattr(record, 'column_number', None),
    getattr(record, 'cause' None))
```

Reply:
@fedde-s @SRodenburg Thanks for the tip